src/component/Navbar/Navbar.js

```diff
@@ -0,0 +1,55 @@
+import './Navbar.css'
+
+import React, {useState} from "react"
+
+import { Anchor, Drawer, Button} from "antd"
+const { Link } = Anchor
+
+const Items = [
```

Comment:
Tą tablicę mógłbyś zamienić na tablicę obiektów. Np, 
{
   href: "#Home",
   title: "Home"
}

Wtedy jest to bardziej czytelne. Gdy będziesz mapował obiekt, dane będziesz wyciągać np tak:
\<Link href={item.href} title={item.title}/>

Wygląda dużo czytelniej